### PR TITLE
doc: add backup recovery page

### PR DESCRIPTION
**Pull request by FrostyX**
SSIA

### CODE REVIEW

**Comment by FrostyX** [comment]
See also https://github.com/fedora-copr/ansible-fedora-copr/pull/3

File: doc/backup_recovery.rst [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+    mkfs.ext4 /dev/md0 -L copr-data
+    mkdir /mnt/data
+    mount /dev/disk/by-label/copr-data /mnt/data/
+
+
+SSH key shenanigans
+...................
+
+Generate a new SSH key::
```
Comment (nikromen) [inline doc/backup_recovery.rst:61]: To make it more clear I would explicitly mention that we want to do it on our local machine... you know, as soon as it's not an exercise, people may do mistakes :D

```suggestion
Generate a new SSH key on your local machine::
```

**Review by nikromen — COMMENTED** [review]
otherwise looks good to me. But I rather leave the approval to Pavel, because I reviewed it mainly from the clarity point of view, not the functionality point of view :)

File: doc/backup_recovery.rst [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+    cat /proc/mdstat
+
+You can see the raid details using::
+
+    mdadm --detail /dev/md0
+
+Format and mount::
+
+    mkfs.ext4 /dev/md0 -L copr-data
```
Comment (xsuchy) [inline doc/backup_recovery.rst:60]: I would add:
tune2fs -m0 /dev/md0
By default it is 5% and that is HUGE amount of data on large volumes.

Comment (FrostyX) [reply doc/backup_recovery.rst:60]: Updated

**Comment by FrostyX** [comment]
I updated the `rsync` command to drop the `--delete` and other unwanted parameters. It finally finished without `malloc` errors.

To sync 20TB of data it took us 7677 minutes (5 days and some change). I haven't yet identified if the bottleneck is the network or disk, I'll need some help with that.

But I suppose we will be addressing the bottleneck in https://github.com/fedora-copr/ansible-fedora-copr/pull/3 , not here.

File: doc/backup_recovery.rst [hunk 3]
```diff
@@ -0,0 +1,130 @@
+.. _backup_recovery:
+
+Recovery from backups
+=====================
+
+In case of emergency ... break glass.
```
Comment (praiskup) [inline doc/backup_recovery.rst:6]: Can we have a rough estimate "what is going to happen now" here?

Comment (FrostyX) [reply doc/backup_recovery.rst:6]: Updated

**Pull request merged**